### PR TITLE
Split PD and PLC factories into separate files.

### PR DESCRIPTION
**Pull request by ashercodeorg**
At over 1K lines, the existing `factories.rb` file was becoming unwieldy. This attempts to reduce its size be splitting apart namespaced factories.

FactoryGirl [documentation](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#defining-factories) indicating that convention will cause these files to automatically be loaded.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Added `README.md` file, per offline request from @aoby.

File: dashboard/test/factories/plc.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+FactoryGirl.allow_class_lookup = false
```
Comment (mehalshah) [inline dashboard/test/factories/plc.rb]: I'd suggest moving PLC into PD. We should have never really made the distinction between the two, and this file is relatively small.

**Comment by aoby** [comment]
After this PR, the file structure will be like this:
```
factories/
  factories.rb # default namespace
  pd.rb # pd namespace
  plc.rb # plc namespace
```

I like this idea of splitting up the factories, and I like the concise file names (DRY). However, I find it a bit confusing to have `factories.rb` vs. `namespace.rb`. I wonder if the naming scheme might be more clear & discoverable this way?

```
factories/
  factories.rb  # default namespace
  pd_factories.rb # pd namespace
  plc_factories.rb
```

File: dashboard/test/factories/plc.rb (hunk 1 above)
Comment (ashercodeorg) [reply dashboard/test/factories/plc.rb]: IMO, that movement should be part of the debt in moving the PLC namespace into the PD namespace, happening at the same time. As long as they are separate namespaces, I feel the factories should be separate.

Comment (aoby) [reply dashboard/test/factories/plc.rb]: In that case we should move those models into the pd namespace. The factory file name should match the model's namespace.

**Comment by ashercodeorg** [comment]
Done (adding `factories` to namespaced filenames).

**Pull request merged**